distribution/helm/vector-aggregator/templates/configmap.yaml

```diff
@@ -6,6 +8,29 @@ metadata:
   labels:
     {{- include "libvector.labels" . | nindent 4 }}
 data:
+  {{- if .Values.customConfig }}
+  {{- if and .Values.customConfig $deprecatedConfig }}{{ fail "Can't use deprecated keys with customConfig" }}{{- end }}
+  vector.yaml: |
+{{ tpl (toYaml .Values.customConfig) . | indent 4 }}
+  {{- else if (not $deprecatedConfig) }}
+  vector.yaml: |
```

Comment:
same here, re: moving this to its own TOML file.

Reply:
replied on the agent/configmap 👍